pyproject.toml

```diff
@@ -2,7 +2,7 @@
 build-backend = 'mesonpy'
 backend-path = ['.']
 requires = [
-  'meson>=0.62.0',
+  'meson>=0.63.0',
```

Comment:
Just a comment that yes this should be fine, and a reminder-to-self that I should bump the version numbers for SciPy